verification/abi.go

```diff
@@ -404,6 +423,18 @@ func (c *dpeABI[CurveParameter, Digest]) CertifyKeyABI(cmd *CertifyKeyReq[Digest
 	}, nil
 }
 
+// Sign calls the DPE Sign command.
+func (c *dpeABI[_, Digest]) SignABI(cmd *SignReq[Digest]) (*SignResp[Digest], error) {
```

Comment:
I think this needs to be rebased with main

Reply:
> I think this needs to be rebased with main

This is rebased can you please review the rebased code?